File: src/main/java/com/tools/properties_settler.java
package com.tools;
import java.io.*;
import java.util.Properties;


public class properties_settler {
    public static void settle() {
        new File("./cache/properties").mkdirs();
        String fileName = "./cache/properties/property.properties";
        try {
            File f = new File(fileName);
            if(!f.exists()){
                f.createNewFile();
                BufferedWriter bw = new BufferedWriter(new FileWriter("./cache/properties/property.properties"));
                bw.write("Log=true\n");
                bw.close();
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
    }

    public static String read(String path, String key) throws IOException {
        InputStream in = new BufferedInputStream(new FileInputStream(path));
        Properties properties = new Properties();
        properties.load(in);
        return (properties.getProperty(key));
    }
}

File: src/main/kotlin/tools/properties_settler.kt
package tools

import java.io.*
import java.util.*

object properties_settler {
    fun settle() {
        File("./cache/properties").mkdirs()
        val fileName = "./cache/properties/property.properties"
        try {
            val f = File(fileName)
            if (!f.exists()) {
                f.createNewFile()
                val bw = BufferedWriter(FileWriter("./cache/properties/property.properties"))
                bw.write("Log=true\n")
                bw.close()
            }
        } catch (e: IOException) {
            e.printStackTrace()
        }
    }

    @Throws(IOException::class)
    fun read(path: String?, key: String?): String {
        val `in`: InputStream = BufferedInputStream(FileInputStream(path))
        val properties = Properties()
        properties.load(`in`)
        return properties.getProperty(key)
    }
}